pkg/client/test_clientset.go

```diff
@@ -48,3 +49,17 @@ func (TestSegmentsClient) Create(ctx context.Context, data []byte) (segments.Res
 func (TestSegmentsClient) Get(ctx context.Context, id string) (segments.Response, error) {
 	return segments.Response{}, fmt.Errorf("unimplemented")
 }
+
+type TestServiceLevelObjectiveClient struct{}
+
+func (TestServiceLevelObjectiveClient) List(ctx context.Context) (api.PagedListResponse, error) {
```

Comment:
Will dry-run still work?

Reply:
This is the test client, the dummy client is used for dry-run. I can rename the dummy_client in a separate PR :D.